pyro/infer/mcmc/api.py

```diff
@@ -412,7 +412,8 @@ def model(data):
             elif self.kernel.model:
```

Comment:
@neerajprad In mp, I think `self.kernel.transforms` will be None, so we need this branch to define `self.transforms`.

Reply:
I see, but with your PR this won't try to find valid initial parameter set again, right?

Reply:
Just getting the transforms should be fast enough.

Reply:
Yes, I think this PR solves all the known issues.

Reply:
That's great! This was actually simpler than what I was expecting. Nicely done.